e2e/helpers/userConfig.ts

```diff
@@ -29,37 +27,22 @@ const generateUserEmail = (prefix: string) => {
 }
 
 type UserConfig = {
-  // used to ensure that dynamically generated tests have static names
-  // ...see https://github.com/microsoft/playwright/issues/24273#issuecomment-1640003150
-  testName: string
-  storageStatePath: string
-  localStorageKey: string
-  user: TestUser
+  [key in UserPrefixes]: TestUser
 }
 
-export const userConfigs: UserConfig[] = [
-  {
-    testName: userNamePrefix,
-    storageStatePath: USER_STORAGE_STATE,
-    localStorageKey: USER_NAME_LOCALSTORAGE_KEY,
```

Comment:
Instead of storing the username in local storage, look up the username using the test user's prefix.